cmd/querybuilder.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/snapp-incubator/crafting-table/internal/querybuilder"
+)
+
+var dialect string
+var table string
+
+var querybuilderCmd = &cobra.Command{
+	Use: "qb",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			log.Fatalln("needs a filename and dialect")
+		}
+		filename := args[0]
+
+		inputFilePath, err := filepath.Abs(filename)
+		if err != nil {
+			panic(err)
+		}
+		pathList := filepath.SplitList(inputFilePath)
+		pathList = pathList[:len(pathList)-1]
+		dir := filepath.Join(pathList...)
+		fset := token.NewFileSet()
```

Comment:
rename the parameter to something more specific and readable. Some of the other parameters' names need to renaming too.